tests/src/Core/Cache/MemcacheCacheDriverTest.php

```diff
@@ -15,24 +18,18 @@ class MemcacheCacheDriverTest extends MemoryCacheTest
 
 	protected function getInstance()
 	{
-		if (class_exists('Memcache')) {
-			try {
-				$this->cache = CacheDriverFactory::create('memcache');
-			} catch (\Exception $exception) {
-				throw new \Exception("Memcache - TestCase failed: " . $exception->getMessage(), $exception->getCode(), $exception);
-			}
-			return $this->cache;
-		} else {
-			$this->markTestSkipped('Memcache driver isn\'t available');
-			return null;
+		try {
+			$this->cache = CacheDriverFactory::create('memcache');
+		} catch (\Exception $exception) {
+			throw new \Exception("Memcache - TestCase failed: " . $exception->getMessage(), $exception->getCode(), $exception);
```

Comment:
While we're at it: is this really necessary? phpunit already outputs the name of the test when an exception is thrown.